docs/framework/whats-new/obsolete-members.md

```diff
@@ -710,8 +710,8 @@ The tables in this article list the type members that are obsolete in the .NET F
   
```

Comment:
line 688 - second xref link should be System.Windows.Forms.AccessibleStates.Valid